packages/chakra-ui-docs/docs/accordion.mdx

```diff
@@ -1 +1,219 @@
+        </AccordionItem>
+        <AccordionItem>
+            <AccordionHeader>
+                <Box flex="1" textAlign="left">
+                Section 2 title
+                </Box>
+                <AccordionIcon />
+            </AccordionHeader>
+            <AccordionPanel :pb="4">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+                veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+                commodo consequat.
+            </AccordionPanel>
+        </AccordionItem>        
+    </Accordion>       
+`}</LiveEditor>
+
+### Styling the expanded state
+
+The `AccordionHeader` component has `aria-expanded` set to `true` or `false` depending on the state of the `AccordionItem`. Use the style prop `_expanded` to style this state.
+
+<br />
+<LiveEditor>{`
+    <Accordion>
+        <AccordionItem>
+            <AccordionHeader>
+            <Box flex="1" textAlign="left">
+                Click me to see a different style
+            </Box>
+            <AccordionIcon />
+            </AccordionHeader>
+            <AccordionPanel>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+            tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+            consequat.
+            </AccordionPanel>
+        </AccordionItem>    
+    </Accordion>
+`}</LiveEditor>
+
+## Accessibility
+
+Pressing `space` or `enter` when focus is on the accordion panel header will
+toggle (expand or collapse) the accordion.
+
+## Props
+
+### Accordion Props
+
+| Name          | Type                              | Default | Description                                                                                               |
+|---------------|-----------------------------------|---------|-----------------------------------------------------------------------------------------------------------|
+| allowMultiple | `boolean`                         |         | If `true`, multiple accordion items can be expanded at once.                                              |
+| allowToggle   | `boolean`                         |         | If `true`, any expanded accordion item can be collapsed again.                                            |
+| index         | `NumberOrArrayOfNumber`           |         | The index(es) of the expanded accordion item                                                              |
+| defaultIndex  | `NumberOrArrayOfNumber`           |         | The initial index(es) of the expanded accordion item                                                      |
+| onChange      | `(NumberOrArrayOfNumber) => void` |         | The callback invoked when accordion items are expanded or collapsed.                                      |
```

Comment:
For all props that have the `onChange` we instead emit the `change` event to the parent context. 

So this should not be here.

Create another table in Markdown called Events and then add `change` as the event that is passed. The payload of the `Accordion` component is a `Number`.

For reference, see how buefy documents events for it's components at https://buefy.org